content/applications/finance/accounting/payables/pay/multiple.rst

```diff
@@ -2,146 +2,43 @@
-recorded.
-
-First, we need to create a payment
-
-This will handle from :menuselection:`Dashboard --> Bank journal -->
-More Option --> Send Money`
-
-.. image:: multiple/multiple07.png
-  :align: center
-
-Creating payment order with check payment method. Selecting related
-Vendor and amount which remain to pay. After filling all details, we
-will confirm the payment order which will generate payment transaction
-with the system.
-
-.. image:: multiple/multiple08.png
-  :align: center
-
-As you can see, bill payment status show what is posted and what is
-remaining to reconcile.
-
-After receiving bank statement from the bank with payment detail, you
-can reconcile the transaction from the Dashboard. It will automatically
-map the transaction amount.
+Odoo offers the possibility to group up multiple bills' payments into one, facilitating the
+reconciliation process.
 
 .. seealso::
+   - :doc:`../supplier_bills/manage`
    - :doc:`../../bank/reconciliation/use_cases`
 
-Partial payments of several supplier bills
-==========================================
-
-How to pay several supplier bills having cash discounts at once?
-----------------------------------------------------------------
-
-You already learned how to pay bills in various way but what about
-partial payment? We are taking another example where we will do partial
-payment for various bills.
-
-We are creating multiple bills and partially pay them through bank
-statements.
-
-We are adding payment terms which allow some cash discount where vendor
-offer us early payment discount.
-
-.. image:: multiple/multiple09.png
-  :align: center
-
-We are creating the following bills with the assignment of the above
-payment term.
-
-.. image:: multiple/multiple10.png
-  :align: center
+Group payments
+==============
 
-We have created the following bills:
+To register the payment of multiple bills at once, go to :menuselection:`Vendors --> Bills`. Then,
+select the bills you wish to register a payment for by **ticking** the boxes next to the bills. When
+done, either click :guilabel:`Register Payment` or :menuselection:`Action --> Register Payment`.
 
-.. image:: multiple/multiple11.png
-  :align: center
+.. image:: multiple/group-payment-action.png
+   :align: center
+   :alt: Action menu and the possible options for bills.
 
-We will pay the invoices by creating bank statement where we will adjust
-the cash discount our vendor provided under payment terms.
+When registering the payments, a **pop-up window** appears. From here, you can either create
+**separate payments** for each bills all at once by clicking :guilabel:`Create Payment`, *or* you
+can create one payment combining **all** bills' payments. To **combine** all payments, tick the
+:guilabel:`Group Payments` box.
 
-.. image:: multiple/multiple12.png
-  :align: center
+.. image:: multiple/multiple-group-payments.png
+   :align: center
+   :alt: Group payments options when registering a payment.
```

Comment:
I don't see the "group payment" option

Reply:
I checked and have 3 different windows on 3 different 14.0 runbots with the same flow, so I asked the POs to know which one is correct one :)

Reply:
> I checked and have 3 different windows on 3 different 14.0 runbots with the same flow, so I asked the POs to know which one is correct one :)

Checked with TSB, it's correct